tests/override_config_table/test_override_config_table_masic.py

```diff
@@ -0,0 +1,234 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import skip_release
+from tests.common.utilities import update_pfcwd_default_state
+from tests.common.config_reload import config_reload
+from utilities import backup_config, restore_config, get_running_config,\
+    reload_minigraph_with_golden_config, file_exists_on_dut
+
+NON_USER_CONFIG_TABLES = ["FLEX_COUNTER_TABLE"]
+
+GOLDEN_CONFIG = "/etc/sonic/golden_config_db.json"
+GOLDEN_CONFIG_BACKUP = "/etc/sonic/golden_config_db.json_before_override"
+CONFIG_DB = "/etc/sonic/config_db.json"
+CONFIG_DB_BACKUP = "/etc/sonic/config_db.json_before_override"
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t2', 't1'),
+    pytest.mark.disable_loganalyzer,
+]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def check_image_version(duthost):
+    """Skips this test if the SONiC image installed on DUT is older than 202111
+
+    Args:
+        duthost: DUT host object.
+
+    Returns:
+        None.
+    """
+    skip_release(duthost, ["201811", "201911", "202012", "202106", "202111"])
+
+
+@pytest.fixture(scope="module")
+def setup_env(duthost, tbinfo):
+    """
+    Setup/teardown
+    Args:
+        duthost: DUT.
+    """
+    topo_type = tbinfo["topo"]["type"]
+    if topo_type in ["m0", "mx"]:
+        original_pfcwd_value = update_pfcwd_default_state(duthost, "/etc/sonic/init_cfg.json", "disable")
+    # Backup configDB
+    for asic_id in duthost.get_asic_ids():
+        config = "/etc/sonic/config_db{}.json".format(asic_id)
```

Comment:
for single box dev, there is no config_dbX.json file

Reply:
filed https://github.com/sonic-net/sonic-mgmt/pull/10522 to resolve it

Reply:
https://github.com/sonic-net/sonic-mgmt/pull/10363/files should have fixed it.